plugins/autocomplete/plugin.js

```diff
@@ -207,6 +217,23 @@
 		 */
 		this._listeners = [];
 
+		/**
+		 * Output template used to write selected match into an editor.
+		 * You can use any parameter passed into `dataCallback` parameter.
```

Comment:
I don't think this is a decent description.

> You can use any parameter passed into `dataCallback` parameter.

So I understand that if I call `dataCallback` with `dataCallback( 1, 2, 3, 4, 5 )` I could use any of these. And by the way, looking only at mentions API, what is `dataCallback`?

Remember this plugin needs to have a simple API.

Please try rewrite this API docs in a simpler way.